libs/styles/src/lib/scss/components/_status-badge.scss

```diff
@@ -13,4 +13,17 @@
   font-size: get-font-size(80);
   line-height: get-line-height(100);
   white-space: nowrap;
+
+  svg {
+    position: absolute;
+    background-color: var(--background-color);
+    fill: var(--color);
+    stroke: var(--color);
+    border-radius: var(--cvi-radius-s);
+    margin-left: 5px;
+
+    path {
```

Comment:
It's preferable to either use an existing "info" icon without changes in the CSS or to add a new icon to `icons` package. Icons should be consistent in design across the library, including small features like stroke widths.